Libs/ClassSpells-1.0/warrior.lua

```diff
@@ -34,14 +35,15 @@ local WarriorClassSpells = {
       rank = 1,
       id = 6343,
       level = 6,
-      cost = 100
+      cost = 100,
+      replaced_by = { 8198, 8204, 8205, 11580 }
     },
     ['parry'] = {
       name = 'Parry',
       rank = 1,
       id = 3127,
       level = 6,
-      cost = 100
+      cost = 100,
```

Comment:
```suggestion
      cost = 100
```
I have been avoiding trailing comma's, i'm not sure how much lua likes them, I know it makes future diffs less clean but I don't mind it :)